types/jest/index.d.ts

```diff
@@ -11,10 +11,9 @@
 //                 Jamie Mason <https://github.com/JamieMason>
 //                 Douglas Duteil <https://github.com/douglasduteil>
 //                 Ahn <https://github.com/AhnpGit>
+//                 Josh Goldberg <https://github.com/joshuakgoldberg>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
```

Comment:
Please put this back, Travis failed because it couldn't find this line.

Reply:
Spooky - must have been a bad merge...